spec/models/passenger_spec.rb

```diff
@@ -38,4 +38,14 @@
       expect(@passenger.temporary?).to eql true
     end
   end
+
+  describe 'self.deactivate_expired_doc_note' do 
```

Comment:
This method depends on the grace period, so if that's changed this test could be affected. It'd be nice for the fact that this method uses an external method call as a data source to be reflected in its test.